packages/react/src/ReactServer.js

```diff
@@ -66,7 +57,6 @@ export {
   startTransition,
   useId,
   useCallback,
-  useContext,
```

Comment:
Breaking change: I'm removing `useContext` from the shared subset since it's never valid in the React Server environment now.